src/Integration/Binding/BindingController.cs

```diff
@@ -138,31 +138,17 @@ void IBindingWorkflowExecutor.BindProject(BindCommandArgs bindingArgs)
 
             var currentConfiguration = configProvider.GetConfiguration();
 
-            SonarLintMode modeToBind;
-
             // If we are currently in standalone then the project is being bound for the first time.
             // Otherwise, we are updating an existing binding
             var isFirstBinding = currentConfiguration.Mode == SonarLintMode.Standalone;
 
-            // TODO: CM cleanup - will never be saving in legacy format
```

Comment:
Essentially the change is to remove this TODO.

I've manually tested that binding/updating still work.